src/main/webapp/mapsScript.js

```diff
@@ -13,8 +13,9 @@
 // limitations under the License.
 
 // Create the script tag, set the appropriate attributes
-var script = document.createElement('script');
-script.src = 'https://maps.googleapis.com/maps/api/js?key=AIzaSyCmQyeeWI_cV0yvh1SuXYGoLej3g_D9NbY&libraries=places&callback=initMap';
+let script = document.createElement('script');
+script.src = 'https://maps.googleapis.com/maps/api/js?key=' + config.API_KEY + 
+  '&libraries=places&callback=initMap';
```

Comment:
why do you not need to wait on initScript here like you do in script.js?

Reply:
This file waits on the `initMap` callback, which is farther down in the file. There were some optional structural decisions here, I could load some of the components without waiting on `initScript`, but I moved everything into one place for simplicity. Happy to talk more on this!